src/app_service/service/wallet_account/service.nim

```diff
@@ -78,7 +78,8 @@ type DerivedAddressesArgs* = ref object of Args
 type TokensPerAccountArgs* = ref object of Args
   accountsTokens*: OrderedTable[string, seq[WalletTokenDto]] # [wallet address, list of tokens]
 
-const CheckBalanceIntervalInMilliseconds = 15 * 60 * 1000 # 15 mins
+const CheckBalanceSlotExecuteIntervalInMilliseconds = 15 * 60 * 1000 # 15 mins
+const CheckBalanceTimerIntervalInMilliseconds = 1000 # 1 sec
```

Comment:
@endulab we should try with a bit larger interval here, maybe 5 or 10 seconds... doing it every second is too often.

Reply:
Set to 5 seconds.